lib/takeaway.rb

```diff
@@ -0,0 +1,18 @@
+class Takeaway
+  def initialize(menu:menu, order:nil)
+    @menu = menu
+    @order = order
+  end
+
+  def dishes_with_prices
```

Comment:
I'd probably make it explicit in the method name that this is a method for displaying/printing.